pages/clustering/replication.mdx

```diff
@@ -126,6 +126,22 @@ The MAIN's UUID is also stored on a disk, in case of restart of an instance to c
 MAIN instance. When REPLICA restarts, `--replication-restore-state-on-startup` must be set to 
 `true` to continue getting updates from the MAIN.
 
+## Auth data replication (Enterprise)
+
+If you are using a Memgraph Enterprise license, all authentication/authorization data, including users, roles, and associated permissions, will be replicated. 
+However, authentication modules are not replicated and must be configured manually by the administrator.
```

Comment:
Please separate this into its own H2 section called "Auth modules replication (Enterprise)"